src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseTagResource.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.digital.ho.hocs.casework.api;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.digital.ho.hocs.casework.api.dto.CaseTagDto;
+import uk.gov.digital.ho.hocs.casework.api.dto.CaseTagRequest;
+
+import java.util.UUID;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+class CaseTagResource {
+
+    private final CaseTagService caseTagService;
+
+    public CaseTagResource(CaseTagService caseTagService) {
+        this.caseTagService = caseTagService;
+    }
+
+    //@Authorised(accessLevel = AccessLevel.WRITE)
```

Comment:
I'd like for us to somehow be able to only allow this from internal services - through a new header maybe thats injected by workflow service?

This would bypass this authorisation check.

Reply:
I think theres a presumption that if a user can progress a case that they have permissions of WRITE or higher.

Reply:
As this is for internal use this does not need an Authorised - we need to look at how we can protect internal APIs.